test/topology_custom/test_raft_no_quorum.py

```diff
@@ -37,6 +37,10 @@ async def test_cannot_add_new_node(manager: ManagerClient) -> None:
             {
                 'name': 'group0-raft-op-timeout-in-ms',
                 'value': '1000'
+            },
+            {
+                'name': 'raft-group-registry-fd-threshold-in-ms',
+                'value': '500'
```

Comment:
Hopefully the test won't be flaky. Let's run it 100 times in debug mode on CI (using the byo_build_test_dtest job)